### ISSUE TITLE
4k option no longer there.

### ISSUE THREAD

**Issue by dvbed** [yuliskov/SmartTube#3695]
My TV is 1080p but on the Smarttube app I've always had the option to choose 4k and it did use to make a difference compared to 1080p whenever I chose 4k. However, since I've updated the app, the 4k option is not there. The 1440p (2k) is also no longer available.

**Comment by gate1975mlm** [comment]
Yep no 4K at all and still get the 403 error message at random times.

YouTube really hates us :(

**Comment by andreferchau** [comment]
Yeah I noticed this on Friday. 1080p is the fastest

**Comment by aksskl** [comment]
I uninstalled latest version and went back to 22.19 stable version, 4k and 1440p are working for me on 22.19. The latest stable version 22.38 does not go above 1080p.

**Comment by Patraputt** [comment]
It's not only 4k, 2k options are missing, VP9 and AV1 are also not available.

**Comment by aksskl** [comment]
I can't test av1 since I'm using a shield, but vp9 4k/60fps works for me using version 22.19

**Comment by andreferchau** [comment]
Thank you!

**Comment by gate1975mlm** [comment]
> I uninstalled latest version and went back to 22.19 stable version, 4k and 1440p are working for me on 22.19. The latest stable version 22.38 does not go above 1080p.

22.19 stable version does not work for me.
